pkg/interop/native/designate/designate.go

```diff
@@ -0,0 +1,25 @@
+package designate
```

Comment:
Better use `roles`, "designate" is kinda obsolete.

Reply:
Fixed.

Reply:
Fixed.